index.html

```diff
@@ -657,6 +657,12 @@
         subscription</a>.
         </li>
       </ol>
+      <p class="note">
+        A <code><a>PushSubscription</a></code> is associated with a <a>service worker
```

Comment:
I think most of this is captured already in http://w3c.github.io/push-api/#push-subscription in normative form, which I prefer over notes, which are non-normative. Maybe just clarify that section by specifically referring to the "clear registration" algorithm?